src/controllers/users.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response } from 'express';
+import { User } from '../entities/users';
+import { UserService } from '../services/UserService';
+import { sendError, sendSuccess } from '../utils/sendResponse';
+
+const userService = new UserService();
```

Comment:
@lognathparamasivam  can you use `dependency-injection` instead of creating the services in the controller using `new`

Reply:
Implemented `dependency-injection` using `tsyring`. Thought to implement basic `constructor based dependency-injection` but gave this a try.